Chap_API_Struct.tex

```diff
@@ -67,7 +67,7 @@ \section{Constants}
 
 \begin{constantdesc}
 %
-\declareconstitem{PMIX_MAX_NSLEN}
+\declareconstitemvalue{PMIX_MAX_NSLEN}{255}
 Maximum namespace string length as an integer.
 \end{constantdesc}
 
```

Comment:
In general, this section is going to be of more interest than the rest as far as the values of constants go.  I don't see any issues, but if we wanted to consider value changes for constants these are the ones to look at.